app/assets/javascripts/angular/components/assignedPeopleSelect/assignedPeopleSelect.component.js

```diff
@@ -33,11 +33,17 @@ function assignedPeopleSelectController(
             assignedPeopleSelectService
                 .searchPeople(search, this.organizationId, requestDeduper)
                 .then(people => {
-                    const assignedPeopleIds = this.assigned.map(({ id }) => id);
-                    // Find all the current people in this.assigned
-                    this.people = people.filter(
-                        person => !assignedPeopleIds.includes(person.id),
-                    );
+                    if (this.assigned === null) {
+                        this.people = people;
+                    } else {
+                        const assignedPeopleIds = this.assigned.map(
```

Comment:
I think you could just do this:
```suggestion
                        const assignedPeopleIds = (this.assigned || []).map(
```

Your solution runs less code if it's null which is good. Writing less code can make things easier to maintain. I'm fine with either and I'm approving what you've got.